node-deb

```diff
@@ -683,25 +683,39 @@ find "$@" -type f -print0 | {
   done
 }
 
-if [ -d "$source_dir/node_modules" ]; then
-  npm ls --parseable --prod | sed -e "s/$(escape "$source_dir")//g" | grep -Ev '^$' | sed -e 's:/node_modules/::g' | {
-    while IFS="\n" read -r -d '' dir; do
-      log_debug "Copying dir: $dir"
-      cp -rf "$source_dir/node_modules/$dir" "$deb_dir/usr/share/$package_name/app/node_modules/"
-    done
-  }
-fi
+app_dir="$deb_dir/usr/share/$package_name/app"
 
-if ! [ -f "$deb_dir/usr/share/$package_name/app/package.json" ]; then
+if ! [ -f "$app_dir/package.json" ]; then
   log_info "Including 'package.json' in the Debian package."
   cp -pf './package.json' "$deb_dir/usr/share/$package_name/app/"
 fi
 
-if [ -f './npm-shrinkwrap.json' ] && ! [ -f "$deb_dir/usr/share/$package_name/app/npm-shrinkwrap.json" ]; then
+if [ -f './npm-shrinkwrap.json' ] && ! [ -f "$app_dir/npm-shrinkwrap.json" ]; then
```

Comment:
If we're using the var `$app_dir` here, we actually might as well get rid of the second `! [ -f $app_dir ]` check since `cp -f ` is force copying it, and we're warning users against including this as a CLI arg themselves. Likewise in the other places.